docs/root/intro/arch_overview/security/bug_bounty.rst

```diff
@@ -0,0 +1,164 @@
+.. _arch_overview_bug_bounty:
+
+Bug bounty
+==========
+
+In the near future, Envoy will have a bug bounty program. This will be open to all security
+researchers and will provide rewards for vulnerabilities discovered and reported according
+to the rules below.
+
+.. note::
+  This is not yet funded, stay tuned and reach out to envoy-security@googlegroups.com for further
+  details.
+
+.. _arch_overview_bug_bounty_rules:
+
+Rules
+-----
+
+The goal of this bug bounty is to provide a formal process to honor contributions from external
+security researchers to Envoy's security. Vulnerabilities should meet the following conditions
+to be eligible for the program:
+
+1. Vulnerabilities must meet one of the below :ref:`objectives
+   <arch_overview_bug_bounty_objectives>`, demonstrated with the supplied Docker-based
+   :ref:`execution environment <arch_overview_bug_bounty_ee>` and be consistent with the
+   program's :ref:`threat model <arch_overview_bug_bounty_threat_model>`.
+
+2. Vulnerabilities must be reported to envoy-security@googlegroups.com and be kept under embargo
+   while triage and potential security releases occur.
+
+3. Vulnerabilities must not be previously known in a public forum, e.g. GitHub issues trackers,
+   CVE databases, etc.
```

Comment:
> CVE databases

Wouldn't it be useful to know if a known CVE applies to Envoy? Or does this mean CVE's reported to Envoy's project

Reply:
Good point, will update.